packages/cdktf/lib/tfExpression.ts

```diff
@@ -131,6 +151,11 @@ export function rawString(str: string): IResolvable {
   return new RawString(str);
 }
 
+// eslint-disable-next-line jsdoc/require-jsdoc
+export function unescapedString(str: string): IResolvable {
+  return new UnescapedString(str);
+}
```

Comment:
This function and the class don't seem to be called / documented or tested anywhere? Is this still required?